compiler/damlc/tests/src/DA/Test/Repl.hs

```diff
@@ -118,7 +117,7 @@ testConnection damlc scriptDar ledgerPort mbTokenFile mbCaCrt = do
         [ "alice <- allocatePartyWithHint \"Alice\" (PartyIdHint \"Alice\")"
         , "debug alice"
         ]
-    let regexString = "^.*daml>.*: 'Alice'\ndaml> Goodbye.\n$" :: String
+    let regexString = "^.*daml>.*: 'Alice[^\n]*\ndaml> Goodbye.\n$" :: String
```

Comment:
```suggestion
    let regexString = "^.*daml>.*: 'Alice:[a-f0-9]+\ndaml> Goodbye.\n$" :: String
```
This does not work ?

Reply:
I'd have to double check, for example I think there would be a close `'` added at least. I'll check